modules/apps/change-tracking/change-tracking-service/source-formatter-suppressions.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+
+<suppressions>
+	<checkstyle>
+		<suppress checks="UnusedMethodCheck" files="src/main/java/com/liferay/change/tracking/internal/configuration/CTConfigurationRegistry\.java" />
```

Comment:
@hhuijser we had to suppress the unused method check, because the private method is being called by the osgi framework but SF don't recognize it.
The way this works is when the `@Reference` is present on a method it's called by OSGi or if the method is being referenced in the same annotation at the `unbind = "_removeCTConfiguration"` attribute. Those are being called by OSGi even if they are not being called in the class anywhere.